README.md

```diff
@@ -1,76 +1,130 @@
+when they're supposed to occur, who's teaching what, and so on.
+
+# Getting started
+
+1. Clone the repository:
+
+    ~~~
+    $ git clone https://github.com/swcarpentry/amy.git
+    $ cd amy
+    ~~~
+
+2. Install Django and its dependencies:
 
     ~~~
     $ sudo python3 -m pip install -r requirements.txt
     ~~~
 
-2. Install [Bower](http://bower.io/) - the tool that manages Amy's JavaScript and CSS dependencies:
+    If you're experienced Python programmer, feel free to create a Python3-compatible virtualenv for Amy and install dependencies from `requirements.txt`.
+
+3. Install [Bower](http://bower.io/) — the tool that manages Amy's JavaScript and CSS dependencies:
 
     ~~~
-    $ npm install -g bower
+    $ sudo npm install -g bower
     ~~~
 
-3.  Setup your local database.  There are two ways you can do this.
+    You may need some additional dependencies to install [Bower](http://bower.io/), such as [NodeJS](https://nodejs.org/) and [npm](https://www.npmjs.com/).
+
+4. Setup your local database.  There are two ways you can do this:
 
     1. If you have access to the legacy data:
 
         1. Create an empty database by running:
 
-           ~~~
-           $ make migrations
-           ~~~
+            ~~~
+            $ make migrations
+            ~~~
 
         2. Fill that database by running:
 
-           ~~~
-           $ make import
-           ~~~
+            ~~~
+            $ make import
+            ~~~
 
-    2. Otherwise set up the redacted data with:
+    2. Otherwise set up the redacted (development-ready) data with:
 
-       ~~~
-       $ make database
-       ~~~
+        ~~~
+        $ make database
+        ~~~
 
-4.  Create a administrator account.
+5. Create an administrator account:
 
     ~~~
     $ python3 manage.py createsuperuser
     ~~~
 
-5.  Start a local Django development server by running (this also installs front-end dependencies for Amy):
+6. Start a local Django development server by running:
 
     ~~~
     $ make serve
     ~~~
 
-6.  Open [http://localhost:8000/workshops/](http://localhost:8000/workshops/) in your browser and start clicking.
+    **Note**:  this also installs front-end dependencies for Amy, such as jQuery or Bootstrap.
+
+7. Open [http://localhost:8000/workshops/](http://localhost:8000/workshops/) in your browser and start clicking.
 
     Use the administrator account that you created.
 
-**Note**: please [check with us](mailto:gvwilson@software-carpentry.org)
-or open a [discussion issue](https://github.com/swcarpentry/amy/labels/discussion)
+# Upgrading
+
+1. Update the code:
+
+    1. Get the list of changes:
+
+        ~~~
+        $ git fetch
+        ~~~
+
+    2. Look for the newest tag:
+
+        ~~~~
+        $ git tag -n
+        ~~~~
+
+    3. Get the code from the newest tag:
+
+        ~~~~
+        $ git checkout tags/<tag_name>
+        ~~~~
+
+2. Update dependencies:
+
+    ~~~
+    $ sudo python3 -m pip install --upgrade -r requirements.txt
```

Comment:
Again, I prefer `python3 -m pip install --user …` to avoid involving root.  I'd rather leave managing system packages to the system's package manager.